addons/mrp/models/stock_move.py

```diff
@@ -565,7 +565,8 @@ def _compute_kit_quantities(self, product_id, kit_qty, kit_bom, filters):
                     continue
                 incoming_moves = bom_line_moves.filtered(filters['incoming_moves'])
                 outgoing_moves = bom_line_moves.filtered(filters['outgoing_moves'])
-                qty_processed = sum(incoming_moves.mapped('product_qty')) - sum(outgoing_moves.mapped('product_qty'))
+                qty_processed = sum(incoming_moves.mapped(lambda m: m.quantity if m.picked else m.product_qty))
```

Comment:
@Whenrow  forgot to mention this change aswell
so far "_compute_kit_quantities" always considers the demand qty 
but if the move is done what really care about is the done qty 
we could use a context key instead like proposed in https://github.com/odoo/odoo/pull/161920